packages/documentation/src/stories/components/tabs/tabs.snapshot.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Args, StoryContext, StoryObj } from '@storybook/web-components';
+import meta from './tabs.stories';
+import { html } from 'lit';
+import { bombArgs } from '../../../utils/bombArgs';
+
+export default {
+  ...meta,
+  title: 'Snapshots',
+};
+
+type Story = StoryObj<HTMLPostTabsElement>;
+
+export const Tabs: Story = {
+  render: (_args: HTMLPostTabsElement, context: StoryContext<HTMLPostTabsElement>) => {
+    return html`
+      <div>
+        ${['bg-white', 'bg-dark'].map(bg => html`
+          <div
+            class="${bg} mt-3"
+            style="padding: 1rem; display: flex; flex-direction: column; gap: 1rem;"
```

Comment:
I think you can use bootstrap here as well

Reply:
We had styles in most snapshot stories, I created PR #1844 to fix this.